src/api/app/policies/package_policy.rb

```diff
@@ -22,6 +22,10 @@ def update?
     user.can_modify?(record)
   end
 
+  def update_labels?
```

Comment:
Why having a different policy method?

Reply:
Because of BsRequests, which aren't covered by this PR

Reply:
Alright...